core/frontend/src/UtilityTypes.ts

```diff
@@ -0,0 +1,18 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
I'd call this file `DeprecatedTypes.ts` so people will know not to include it in frontend code and that it will go away in 4.0.

Reply:
It needs a packageDocumentation tag. Not all deprecated types will be in the same documentation group. People who include it in frontend code will be told by linter that they are using a deprecated type. I don't want to create a documentation group called "deprecated stuff" in our published docs.

Reply:
i'd still rename it so no one adds anything else to it.

Reply:
done.